src/rendering/renderers/shared/Renderable.ts

```diff
@@ -8,12 +8,12 @@ export interface Renderable<VIEW extends View = View> extends EventEmitter
     uid: number;
     view: VIEW;
     didViewUpdate: boolean;
-    layerTransform: Matrix;
+    renderGroupTransform: Matrix;
```

Comment:
for consistency should this be `rgTransform`?

Reply:
yes!